Library/Homebrew/livecheck.rb

```diff
@@ -104,6 +128,7 @@ def to_hash
       "skip_msg" => @skip_msg,
       "strategy" => @strategy,
       "url"      => @url,
+      "version"  => @version,
```

Comment:
Would we want to omit this from the hash unless `@formula_or_cask` is a cask?

Reply:
`@version` should be initialized to `nil` like the others. We include values in the hash for instance variables that haven't been set by the `livecheck` block, so `@version` is no different and should be included in the `to_hash` hash.